undefined: cancel error

I think PR #22 broke the build and now chainstore can't be installed.

```
go get -u github.com/pressly/chainstore
# github.com/pressly/chainstore
../../pressly/chainstore/timeout.go:25:7: undefined: cancel
../../pressly/chainstore/timeout.go:26:8: undefined: cancel
../../pressly/chainstore/timeout.go:31:7: undefined: cancel
../../pressly/chainstore/timeout.go:32:8: undefined: cancel
../../pressly/chainstore/timeout.go:37:7: undefined: cancel
../../pressly/chainstore/timeout.go:38:8: undefined: cancel
```

--- a/timeout.go
+++ b/timeout.go
@@ -22,19 +22,19 @@ func (s *timeoutManager) Open() (err error)  { return s.chain.Open() }
 func (s *timeoutManager) Close() (err error) { return s.chain.Close() }
 
 func (s *timeoutManager) Put(ctx context.Context, key string, val []byte) (err error) {
-	ctx, cancel = context.WithTimeout(ctx, s.timeout)
+	ctx, cancel := context.WithTimeout(ctx, s.timeout)
 	defer cancel()
 	return s.chain.Put(ctx, key, val)
 }
 
 func (s *timeoutManager) Get(ctx context.Context, key string) (data []byte, err error) {
-	ctx, cancel = context.WithTimeout(ctx, s.timeout)
+	ctx, cancel := context.WithTimeout(ctx, s.timeout)
 	defer cancel()
 	return s.chain.Get(ctx, key)
 }
 
 func (s *timeoutManager) Del(ctx context.Context, key string) (err error) {
-	ctx, cancel = context.WithTimeout(ctx, s.timeout)
+	ctx, cancel := context.WithTimeout(ctx, s.timeout)
 	defer cancel()
 	return s.chain.Del(ctx, key)
 

